common/src/main/java/com/microsoft/hydralab/performance/parsers/IOSEnergyGaugeResultParser.java

```diff
@@ -74,6 +74,8 @@ public PerformanceTestResult parse(PerformanceTestResult performanceTestResult)
                         lineNumber++;
                     }
                     performanceTestResult.performanceInspectionResults = newPerfInspectionResults;
+//                    performanceTestResult.setResultSummary(getSumIOSEnergy(newPerfInspectionResults));
```

Comment:
This line of code is no longer needed?

Reply:
Syned with @dinghong2233, we may keep both summary types of `SUM` and `AVERAGE`. I added a comment to explain it.